packages/editor-ui/src/views/NodeView.vue

```diff
@@ -3771,21 +3779,36 @@ export default defineComponent({
 			nodeTypes: Array<{ nodeTypeName: string; position: XYPosition }>,
 			dragAndDrop: boolean,
 		) {
-			nodeTypes.forEach(({ nodeTypeName, position }, index) => {
-				const isManualTrigger = nodeTypeName === MANUAL_TRIGGER_NODE_TYPE;
-				const openNDV = !isManualTrigger && (nodeTypes.length === 1 || index > 0);
+			const enrichedNodeTypes = nodeTypes
```

Comment:
would be great if we did not make NodeView any bigger.. could we refactor this somehow outside of this.. for example this could be a util function under `nodeViewUtils`
then it's also testable as a separate function